### PR TITLE
Changes to classifier

### LINKED ISSUE #6: [Classification] Set up SVM

### PR DESCRIPTION
**Pull request by VictorSuciu**
updated ModelData class
moved BERT embedding code to its own file
Added SVM training and validation
Closes #6

### CODE REVIEW

File: src/classifier.py [hunk 1]
```diff
@@ -1,67 +1,38 @@
...
+# These will be the list of labels called like so:
+#
+# map_labels = Label.maps(all_problems)
+# list_labels = Label.loops(all_problems)
+# ...
+# =================================
+map_labels = []
+list_labels = []
+loop_labels = []
+nested_loop_labes = []
```
Comment (isabelgiang) [inline src/classifier.py:21]: should this be `nested_loop_labels`?

**Review by isabelgiang — APPROVED** [review]
I would delete the `__pycache__` files from being tracked by Git so there's less clutter in the repo, but other than that, LGTM

File: src/classifier.py (hunk 1 above)
Comment (VictorSuciu) [reply src/classifier.py:21]: I'm not quite sure what you mean by that.
I'll add a .gitignore rn

Comment (VictorSuciu) [reply src/classifier.py:21]: oh yes oops it's mispelled

**Pull request merged**